README.md

```diff
@@ -59,17 +59,20 @@ Note: Use HTTP Basic Auth for these APIs (Username/password is the credentials f
 -H 'Authorization: Basic c2VydmljZXMtYWRtaW46cGFzc3dvcmQ=' \
 -d '{"name": "test8"}'`
 #### Update a WSDL Doc Service's Operations by URL of WSDL resource
-`curl -X PUT [your-website-url]/[your-service-path-to-endpoint]/wsdl/[wsdl-doc-name] \
+`curl -X PUT [your-website-url]/[your-service-path-to-endpoint]/wsdl/[wsdl-doc-machine-name] \
 -H 'Authorization: Basic c2VydmljZXMtYWRtaW46cGFzc3dvcmQ=' \
 -H 'Content-Type: application/json' \
 -d '{"url": "https://graphical.weather.gov/xml/DWMLgen/wsdl/ndfdXML.wsdl"}'`
-#### Import a WSDL to an Existing Service by WSDL file
-`curl -X POST [your-website-url]/[your-service-path-to-endpoint]/wsdl/[wsdl doc name]/import \
+#### Import a WSDL file(s) to an Existing Service by WSDL file
+This example shows how to import 2 files. You may import as many as you like, just be sure to increment 'x' for each file: soap_api_definition[x]. The first file i.e. soap_api_definition[0] should be the main file that makes any references to additional files. 
+
+`curl -X POST [your-website-url]/[your-service-path-to-endpoint]/wsdl/[wsdl-doc-machine-name]/import \
 -H 'Authorization: Basic c2VydmljZXMtYWRtaW46cGFzc3dvcmQ=' \
 -H 'content-type: multipart/form-data' \
--F 'soap_api_definition=@[path-to-wsdl]'`
+-F 'soap_api_definition[0]=@[path-to-wsdl-1]'
```

Comment:
Using a series of `soap_api_definition[]` doesn't work? its kinda ugly to have to increment an index, but if that's the only option then ok.

Reply:
No you totally don't need to do that. Changed it.